public/src/modules/navigator.js

```diff
@@ -389,7 +368,7 @@ define('navigator', ['forum/pagination', 'components'], function (pagination, co
 		var navbarHeight = components.get('navbar').height();
 
 		// Temporarily disable navigator update on scroll
-		$(window).off('scroll', navigator.update);
+		$(window).off('scroll', navigator.delayedUpdate);
```

Comment:
Seems @barisusakli fixed some bug (in b84f36cc201c61cc65af85729eea6e7afe75d2d0) by introducing `navigator.delayedUpdate`, but this invocation was not updated. It caused multiple events to be added the more the navigator was interacted with.